edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_dashboard.scss

```
@@ -0,0 +1,463 @@
+// Change cards from landscape to portrait
+
+.view-dashboard div.item .items.course.single-course {
+  float: left;
+  margin: 1%;
+  min-width: 250px;
+  padding: 0px;
+  width: 23%;
+}
+
+@media (max-width: 1190px) {
+  .view-dashboard div.item .items.course.single-course {
+    width: 31%;
+  }
+}
+
+@media (max-width: 890px) {
+  .view-dashboard div.item .items.course.single-course {
+    width: 48%;
+  }
+}
+
+@media (max-width: 660px) {
+  .view-dashboard div.item .items.course.single-course {
+  width: 98%;
+  }
+}
+
+.view-dashboard div.item li.course-item {
+  text-align: left;
+  width: 100%;
+  box-shadow: 0 1px 10px 0 rgba(0,48,87,0.125), inset 0 0 0 1px rgba(255,255,255,0.75);
+}
+
+.view-dashboard div.item li.course-item:hover {
+  box-shadow: 0 1px 4px 0 rgba(0,127,163,0.4);
+}
+
+@media (max-width: 991px) {
+  .view-dashboard div.item .course .details {
+    padding-left: 0px !important;
```

```suggestion
    padding-left: 0 !important;
```

Same suggestion about px unit